src/tests/JIT/Regression/JitBlue/Runtime_105484/Runtime_105484.cs

```diff
@@ -0,0 +1,54 @@
+// Generated by Fuzzlyn v2.2 on 2024-07-30 17:57:54
+// Run on Arm64 Windows
+// Seed: 1755918021774833910-vectort,vector64,vector128,armsve
+// Reduced from 111.7 KiB to 1.0 KiB in 00:00:56
+// Hits JIT assert in Release:
+// Assertion failed '!IsUnusedValue()' in 'Program:Main(Fuzzlyn.ExecutionServer.IRuntime)' during 'Lowering nodeinfo' (IL size 108; hash 0xade6b36b; FullOpts)
+//
+//     File: C:\dev\dotnet\runtime\src\coreclr\jit\gentree.cpp Line: 18154
+//
+using System;
+using System.Runtime.CompilerServices;
+using System.Numerics;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+
+public class Runtime_105484
+{
+    public static byte[] s_1;
+    public static byte[,, ] s_2;
+    public static Vector<uint> s_5;
+
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    public static void M()
```

Comment:
This should be private (I think this fails to build due to the test analyzers we have)